src/monodroid/jni/cppcompat.h

```diff
@@ -20,5 +20,31 @@ namespace std
 	{
 		return static_cast<typename remove_reference<decltype(arg)>::type&&>(arg);
 	}
+
+	template<typename TMutex>
+	class lock_guard
+	{
+	public:
+		using mutex_type = TMutex;
+
+	public:
+		lock_guard (const lock_guard&) = delete;
```

Comment:
Oh, so `= delete` is the newfangled way to remove a method?

I'm going to need a guide for the C++ changes since C++-98.  Have a handy link to one?

Reply:
This should introduce you to [the basics](https://www.amazon.com/Tour-2nd-Depth-Bjarne-Stroustrup/dp/0134997832/ref=sr_1_1?ie=UTF8&qid=1544629378&sr=8-1&keywords=a+tour+of+c%2B%2B%2C+2nd+edition) and the Wikipedia article [on C++11](https://en.wikipedia.org/wiki/C%2B%2B11) is not bad either